cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -59,6 +59,37 @@ describe('Creator Settings Page', () => {
     cy.url().should('equal', baseUrl);
   });
 
+  it('should update the colors on the form when a new brand color is selected', () => {
+    const color = '#25544b';
+    const rgbColor = 'rgb(37, 84, 75)';
+
+    cy.findByLabelText(/^Brand color/)
+      .clear()
+      .type(color)
+      .trigger('change');
+
+    cy.findByRole('button', { name: 'Finish' }).should(
+      'have.css',
+      'background-color',
+      rgbColor,
+    );
+
+    cy.findAllByRole('radio', { name: /members only/i })
+      .check()
+      .should('have.css', 'background-color', rgbColor)
+      .should('have.css', 'border-color', rgbColor);
+
+    cy.findByRole('textbox', { name: /community name/i })
+      .focus()
+      .should('have.css', 'border-color', rgbColor);
+
+    cy.get('.admin-help-button').should(
```

Comment:
Refactor (blocking) use `cy.findByRole instead as it's more reliable than a CSS class name which may change in the future.

```suggestion
    cy.findByRole('link', { name: /community name/i }).should(
```